ex02.ex

```diff
@@ -41,7 +43,7 @@ defmodule Ex02 do
   # Do the same using the & syntax #
   ##################################
 
-  list2b = your_anonymous_function(1, 2)
+  list2b = &([(&1 + &2), (&1 - &2)])
```

Comment:
No deductions, but I think the form without the unneeded parens is easier to read:

~~~ elixir
list2b = &[ &1 + &2, &1 - &2 ]
~~~